libs/positions/src/lib/positions-manager.tsx

```diff
@@ -25,7 +25,13 @@ export const PositionsManager = ({ partyId }: PositionsManagerProps) => {
 
   return (
     <>
-      <AsyncRenderer loading={loading} error={error} data={data}>
+      <AsyncRenderer
```

Comment:
it was displaying `No data` and it was inconsistent with the other tabs. (regression)